README.md

```diff
@@ -7,25 +7,51 @@ Latest build: https://metamask.github.io/api-specs/latest/openrpc.json
 
 JSON-RPC API provided by MetaMask.
 
-MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human-readable and machine-readable. It will improve the accuracy of documentation, API, and clients.
+MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human- and machine-readable. It will improve the accuracy of documentation, API, and clients.
```

Comment:
```suggestion
Previously, MetaMask dapp developers lacked a formal RPC specification. This repository contains a specification that documents which methods MetaMask supports in a way that's versionable, human-readable, and machine-readable. This spec improves the accuracy of documentation, API, and clients.
```

Suggestion. `versionable, human-readable, and machine-readable` makes more sense because there are more than two items in the list.